benchmarks/kepler_propagation.py

```diff
@@ -0,0 +1,17 @@
+from poliastro.bodies import Sun, Earth
+from poliastro.twobody import Orbit
+from poliastro.twobody.propagation import cowell, kepler, mean_motion
+
+from astropy import units as u
+
+def time_propagation(method, ecc):
+    a = 7000 * u.km
+    _a = 0.0 * u.rad
+    if ecc < 1.0:
```

Comment:
Now that I see this, I'm not supper happy with the "discontinuity" of the `Orbit.from_classical` function (which we were already aware of anyway, see https://github.com/poliastro/poliastro/wiki/Parabolic-orbits:-final-decision). On the other hand, this test is backwards compatible no matter we decide regarding this particular point.